### PR TITLE
chore(): update ember-cli to 2.4.2

### LINKED ISSUE #315: Update version of the ember-cli to the latest version
Current version of the ember-cli is 2.4.2 and our basis is 1.13.15

### PR DESCRIPTION
**Pull request by jkuri**
Closes #315. Tested most of the commands and it works.

### CODE REVIEW

**Comment by hansl** [comment]
> This branch is out-of-date with the base branch

Fix please.

**Comment by jkuri** [comment]
Fixed.

File: tests/acceptance/init.spec.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ describe('Acceptance: ng init', function() {
         });
 
         expected.forEach(function (file, index) {
-            expected[index] = file.replace(/__name__/g, 'angular-cli');
+            expected[index] = file.replace(/__name__/g, 'tmp');
```
Comment (zackarychapple) [inline tests/acceptance/init.spec.js]: I was struggling with getting the tests to pass when I was bumping versions. I see you changed the expectation, does that mean that the API for the ember-cli changed?

Comment (jkuri) [inline tests/acceptance/init.spec.js]: Not sure about that, but seems like it.

Comment (zackarychapple) [inline tests/acceptance/init.spec.js]: If the API changed shouldn't we switch the implementation rather than changing the test criteria?

Comment (jkuri) [inline tests/acceptance/init.spec.js]: You are right with that.

File: tests/acceptance/init.spec.js [hunk 2]
```diff
@@ -114,7 +114,9 @@ describe('Acceptance: ng init', function() {
                 return ng([
                     'init',
                     '--skip-npm',
-                    '--skip-bower'
+                    '--skip-bower',
+                    '--name',
+                    'tmp'
```
Comment (jkuri) [inline tests/acceptance/init.spec.js]: @zackarychapple I think it is okay. Can you re-check please?

File: tests/acceptance/init.spec.js (hunk 1 above)
Comment (filipesilva) [inline tests/acceptance/init.spec.js]: As far as I can tell, the API hasn't changed per se. It just so happens that the init command on `ember-cli` doesn't use any variables (like `__name__`) on it's blueprints for file paths, whereas we do.

So in line 119 we're using an explicit name (where before we weren't) and here in the test we're manually replacing the `__name__` template string.

It just seems that the test before defaulted to `angular-cli` when no name was given.

**Comment by hansl** [comment]
LGTM. Please rebase, wait for the CI to give you a green flag then you can submit :)  Good work!

**Comment by jkuri** [comment]
Thanks. Merging.

**Pull request merged**